admin/basecalendar-admin.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * BaseCalendar admin
+ *
+ * @package WordPress
+ * @subpackage Combined Calendar
+ * @since 1.0.0
+ * @version 1.0.0
+ */
+
+
+/**
+ * general_setting_page
+ */
+function general_setting_page() { ?>
+	<div class="wrap">
+		<h2>設定</h2>
+		<from method="post" action="options.php" novalidate="novalidate">
+		</from>
+	</div>
+<?php
```

Comment:
I don't prefer close and re-open php tag... 
What do you think about the following?

```php
function xxx () {
    $html = '<div class="wrap">sample text</div>';
    return $html;
}

```